homeassistant/components/dexcom/strings.json

```diff
@@ -15,9 +15,6 @@
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
-    },
-    "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
```

Comment:
`"already_configured"` not used in 
homeassistant/components/dexcom/config_flow.py
tests/dexcom/test_config_flow.py